checker/src/org/checkerframework/checker/index/samelen/SameLenTransfer.java

```diff
@@ -188,14 +179,18 @@ AnnotationMirror getAnno(Node n) {
             CFValue firstValue,
             CFValue secondValue,
             boolean notEqualTo) {
-
-        if (notEqualTo) {
-            // Refinement in the else store if this is a.length != b.length.
-            refineEq(firstNode, secondNode, result.getElseStore());
-        } else {
-            // Refinement in the then store if this is a.length == b.length.
-            refineEq(firstNode, secondNode, result.getThenStore());
+        CFStore equal = notEqualTo ? result.getElseStore() : result.getThenStore();
```

Comment:
Can we rename this to `equalStore` is something else that indicates its a store? The code below is difficult to read with it named `equal`